client/src/components/index.js

```diff
@@ -1,8 +1,8 @@
-export {MainPage} from './Mainpage/MainPage'
-export {Products} from './Products/Products'
-export {ProductDetails} from './Products/ProductDetails'
-export {ProductCard} from './ProductCard/ProductCard'
-export {Modal, Mdl} from './Modal/Modal'
-export {NotFound} from './Notfound/NotFound'
-export {Title} from './Title/Title'
-export {Favourites} from './Favourites/Favourites'
\ No newline at end of file
+export { MainPage } from './Mainpage/MainPage';
+export { Modal, Mdl } from './Modal/Modal';
+export { Products } from './Products/Products';
+export { ProductDetails } from './Products/ProductDetails';
+export { ProductCard } from './ProductCard/ProductCard';
+export { NotFound } from './Notfound/NotFound';
+export { Title } from './Title/Title';
+export { Favorites } from './Favorites/Favorites';
```

Comment:
it's necessary to change it with the line: 
export { Favourites } from './Favourites/Favourites'

pls check it out at develop branch.